tests/sql/test_temporal_exprs.py

```diff
@@ -49,6 +54,11 @@ def test_temporals():
         microsecond(datetimes) as microsecond,
         nanosecond(datetimes) as nanosecond,
         year(datetimes) as year,
+        to_unix_epoch(datetimes) as to_unix_epoch,
+        to_unix_epoch(datetimes, 's') as to_unix_epoch_s,
+        to_unix_epoch(datetimes, 'ms') as to_unix_epoch_ms,
+        to_unix_epoch(datetimes, 'us') as to_unix_epoch_us,
+        to_unix_epoch(datetimes, 'ns') as to_unix_epoch_ns
```

Comment:
Do we support full names to like `seconds`, `milliseconds`, etc?

Reply:
i believe the `TimeUnit.from_str` supports full names like `second` and `millisecond`

Reply:
Sweet, let's test it and possible lock down to what we want to support / document.